lang/en/filter_wiris.php

```diff
@@ -40,14 +40,12 @@
 $string['clearcachedesc'] = 'Clear MathType filter cache';
 $string['clickwirisplugincorrectlyinstalled'] = 'Click the following button to test if MathType is correctly installed.';
 $string['clickwirisquizzescorrectlyinstalled'] = 'Click the following button to test if WirisQuizzes is correctly installed.';
-$string['contact'] = 'For more information or if you have any doubt contact WIRIS Support:';
+$string['contact'] = 'For more information see <a href="https://docs.wiris.com/mathtype/en/mathtype-for-lms/mathtype-for-moodle.html">our documentation </a> or contact WIRIS Support:';
```

Comment:
Maybe this will conflict when changing the Moodle language. I'll check what happens when it is in another one and let you know!

Reply:
Sure! I found that the language has a cache, so if the language is not updated, the cache needs to be purged. I will add this step in the "steps to reproduce." But how can we make users do it?